### ISSUE TITLE
Add igor support

### ISSUE THREAD

**Issue by domna** [FAIRmat-NFDI/pynxtools-mpes#2]
The reader should also support igor binarywave files

**Comment by lukaspie** [comment]
Hey @domna, I just received just some XPS files from a user that are also igor binarywave files. Maybe we could also provide a generic igor parser within https://github.com/FAIRmat-NFDI/pynxtools/pull/250 which could be specialized by the plugins?

**Comment by domna** [comment]
Yes, we could just add a standard parsing function for igor files as we will do with json and yaml files. I don't know generally how uniform the data is stored in the format but I think we will be able to figure something out.

**Comment by rettigl** [comment]
Well, igor waves contain matrix data up to 4 dimensions, with attached unit scaling to each dimension (start, step, unit). Additionally, there is a wavenote, which can contain text. 
Typically, the wave not is being used to store metadata, but the format of this is undefined (often key=value\r\n format).
